File: src/main/java/server/entities/Comportamento.java
package server.entities;

import javax.persistence.Entity;

@Entity
public class Comportamento extends Elemento {

	private static final long serialVersionUID = 1L;

	public Comportamento(String name, String descricao, String cadastradoPor, String complemento) {
		super(name, descricao, cadastradoPor, complemento);
	}
	
	public Comportamento(Elemento elemento) {
		super(elemento.getNome(), elemento.getDescricao(), elemento.getCadastradoPor(), elemento.getComplemento());
	}

	public Comportamento() {
	}

}

File: SeiSaude-app/src/models/comportamento.ts
export class Comportamento {
    nome: string;
    descricao: string;
    cadastradoPor: string;

    public constructor(nome, descricao, cadastradoPor) {
        this.nome = nome;
        this.descricao = descricao;
        this.cadastradoPor = cadastradoPor;
    }
}